src/Sulu/Bundle/SnippetBundle/Resources/public/js/components/webspace/settings/snippets/main.js

```diff
@@ -13,8 +13,8 @@ define(['config', 'app-config'], function(Config, AppConfig) {
 
     var defaults = {
         options: {
-            snippetTypesUrl: '/admin/api/snippet-types?defaults=true&webspace=<%= webspace %>',
-            snippetTypeDefaultUrl: '/admin/api/snippet-types/<%= type %>/default?webspace=<%= webspace %>',
+            snippetAreasUrl: '/admin/api/snippet-types/areas?webspace=<%= webspace %>',
```

Comment:
That's actually not a nice REST URL... This indicates that `areas` is a subresource of `snippet-types`, which again is a collection... maybe `/admin/api/snippet-areas`? I am really not sure about this, maybe we should also ask @chirimoya about his opinion...